orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/DeployManifestContext.java

```diff
@@ -37,12 +40,42 @@ public DeployManifestContext(
     @JsonProperty("manifestArtifactId") String manifestArtifactId,
     @JsonProperty("manifestArtifactAccount") String manifestArtifactAccount,
     @JsonProperty("skipExpressionEvaluation") Boolean skipExpressionEvaluation,
+    @JsonProperty("trafficManagement") TrafficManagement trafficManagement,
     @JsonProperty("requiredArtifactIds") List<String> requiredArtifactIds
   ){
     this.source = source;
     this.manifestArtifactId = manifestArtifactId;
     this.manifestArtifactAccount = manifestArtifactAccount;
     this.skipExpressionEvaluation = skipExpressionEvaluation;
+    this.trafficManagement = trafficManagement;
     this.requiredArtifactIds = requiredArtifactIds;
   }
+
+  @Getter
```

Comment:
Is there a reason to do `@Getter` and `@JsonProperty` rather than `@Data`?

Reply:
Immutability---as we create this from the input map and never modify it (and actually modifications will never be persisted) I didn't want to expose setters.

Reply:
Got it -- SGTM.